### PR TITLE
(#6581): v2 docs building apps with gatsby overview update

### PR DESCRIPTION
**Pull request by amberleyromo**
Tweaks reviewing overview section for v2. Consistency in the page title/sidebar.

### CODE REVIEW

File: docs/docs/building-apps-with-gatsby.md [hunk 1]
```diff
@@ -2,15 +2,18 @@
...
+Gatsby is an excellent framework for building web apps. You can use Gatsby to create personalized, logged-in experiences with two different approaches.
 
-The first approach is to build "hybrid" app pages which are statically rendered with dynamic sections. The second is, if needed, add client-only multi-page sections of the site.
+1. "hybrid" app pages, and
+2. client-only routes & user authentication
 
 ## Hybrid app pages
 
-With this method, Gatsby renders the initial page with shared page content -- then when your React components load in the browser, they can fetch and render data from APIs. The [React docs have a simple example of how to do this.](https://reactjs.org/docs/faq-ajax.html)
+With this method, Gatsby initially renders the page statically, and then when your React components load in the browser they can fetch and render data from APIs.
```
Comment (KyleAMathews) [inline docs/docs/building-apps-with-gatsby.md]: This is kinda awkward language — perhaps change it to describe how a visitor to a Gatsby site first loads an HTML file and then the JavaScript.

**Comment by KyleAMathews** [comment]
Deploy preview for *using-postcss-sass* failed.

Built with commit ebece85ff54155cc1881f080a47eb0f75ded52cc

https://app.netlify.com/sites/using-postcss-sass/deploys/5b69c0d31f12b768e19b3c86

File: docs/docs/building-apps-with-gatsby.md (hunk 1 above)
Comment (amberleyromo) [reply docs/docs/building-apps-with-gatsby.md]: Modified the language

**Comment by KyleAMathews** [comment]
Thanks!

**Pull request merged**